### DISCUSSION TITLE
How to setup mailhog in local

### DISCUSSION THREAD

**Discussion by mavenbird** [wardenenv/warden#701] [Help & Support]
I want to setup Mailhog on my local , how to setup it ?

**Comment by sszymkowiczpg** [comment]
Setup SMTP for Mailhog to catch all the emails:

Host: mailhog
Port: 1025
Authentication: None

Mageplaza SMTP module configuration example for Magento 2
<img width="635" alt="image" src="https://github.com/wardenenv/warden/assets/70631312/d4102d24-89b2-4bda-a544-ac9d585755d8">
